assets/js/components/higherorder/withdata.js

```diff
@@ -242,14 +243,19 @@ const withData = (
 
 			const moduleName = module ? googlesitekit.modules[ module ].name : __( 'Site Kit', 'google-site-kit' );
 
+			// If module is active but setup not complete.
+			if ( module && googlesitekit.modules[ module ].active && ! googlesitekit.modules[ module ].setupComplete ) {
+				return getSetupIncompleteComponent( moduleName, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid );
+			}
+
 			// If we have an error, display the DataErrorComponent.
 			if ( error ) {
 				return ( 'string' !== typeof error ) ? error : getDataErrorComponent( moduleName, error, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid );
 			}
 
 			// If we have zeroData, display the NoDataComponent.
 			if ( zeroData ) {
-				return getNoDataComponent( moduleName, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid );
+				return getNoDataComponent( moduleName, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid, module );
```

Comment:
The `module` parameter should be removed from here, likely an oversight.

Reply:
removed https://github.com/google/site-kit-wp/pull/84/commits/3308d8b3ac5b53cb45374bfe21334713cd801e50